components/map/events.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Events
+page_title: Map - Events
+description: Discover the Blazor Map events and explore the examples.
+slug: components/map/events
+tags: telerik,blazor,map,events,event
+published: true
+position: 11
+---
+
+# Map Events
+
+This article explains the available events for the Telerik Map for Blazor:
+
+* [OnClick](#onclick)
+* [OnMarkerClick](#onmarkerclick)
+* [OnShapeClick](#onshapeclick)
+
+>important The **asynchronous** rendering is **not** supported.
```

Comment:
This might be confusing - we haven't explained what "async rendering" means in the context of the map. I suggest removing this sentence as this is more of an implementation detail (limitation) for now.

Reply:
Done.